easybuild/easyconfigs/d/DistributedStream/DistributedStream-20210628-gompi-2021a.eb

```diff
@@ -0,0 +1,34 @@
+easyblock = 'MakeCp'
+
+name = 'DistributedStream'
+version = '20210628'
+local_commit = '998f747'
```

Comment:
@adrianjhpc Do you have any plans to tag a "proper" versioned release for `DistributedStream`, or should we keep using a specific commit and resort to a date stamp as "version" (like I'm doing here)?

Reply:
Let me do a proper version release.

Reply:
Release version 1.0 should now be available. Let me know if there are changes  I could make to integrate better into easybuild. Currently it's just a rubbish makefile, so if that doesn't integrate well let me know.

Reply:
:+1: